assets/src/components/v2/departures/header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from "react";
+import cx from "classnames";
+import { imagePath } from "Util/util";
+
+type CardinalDirection = "n" | "ne" | "e" | "se" | "s" | "sw" | "w" | "nw";
+
+interface Props {
+  title: string | null;
+  direction: CardinalDirection | null;
+}
+
+const DirectionArrow = ({ direction }: { direction: CardinalDirection }) => {
+  const src = useMemo(() => imagePath("arrow-45.svg"), []);
+  return (
+    <img
+      src={src}
+      data-direction={direction}
+      className={cx("direction-arrow", {
+        "rotate-45": direction === "n",
+        "rotate-90": direction === "ne",
+        "rotate-135": direction === "e",
+        "rotate-180": direction === "se",
+        "rotate-225": direction === "s",
+        "rotate-270": direction === "sw",
+        "rotate-315": direction === "w",
+      })}
```

Comment:
**note:** I've pulled in `classnames` because it makes handling dynamic classes for components much easier. If there's opposition to introducing this library I'm happy to do this by hand.

Reply:
[Jon would be happy](https://app.asana.com/0/1185117109217422/1201783781811458) 😄